src/steamship/agents/llms/replicate.py

```diff
@@ -0,0 +1,79 @@
+from typing import List, Optional
+
+from steamship import Block, PluginInstance, Steamship
+from steamship.agents.schema import LLM
+
+PLUGIN_HANDLE = "replicate-llm"
+DEFAULT_MAX_TOKENS = 256
+
+LLAMA_2_70B = "replicate/llama-2-70b-chat"
+DOLLY_2_12B = "replicate/dolly-v2-12b"
+
+DEFAULT_MODEL = LLAMA_2_70B
+
+
+class Replicate(LLM):
+    """LLM that uses Steamship's Replicate plugin to generate completions.
+
+    The default model is Llama 2 70B.
+
+    See the Replicate plugin source code for more information:
+      https://github.com/steamship-plugins/replicate-llms
+    """
+
+    generator: PluginInstance
+    client: Steamship
+
+    def __init__(
+        self,
+        client,
+        model_name: str = LLAMA_2_70B,
```

Comment:
The plugin provides its own default here, so it's not strictly necessary to provide one (although whether the plugin should be opinionated about defaults rather than the agent code is up for debate)

ETA: replicate LLMs provide their own defaults across the board for parameters like max_tokens and the like as well, though if we've previously been providing e.g. 0.4 for temperatures that could result in a change in behavior for people moving over -- replicate's default for that field is .75.